src/components/redux/books/books.js

```diff
@@ -0,0 +1,26 @@
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
+
```

Comment:
- [ ] Kindly use the ducks pattern for your action type values.

  ```
  const ADD_BOOK = 'bookstore/books/ADD_BOOK';
  const REMOVE_BOOK = 'bookstore/books/REMOVE_BOOK';
  ```

  You can look at [this article](https://medium.com/@matthew.holman/what-is-redux-ducks-46bcb1ad04b7) for more information